app/views/hosts/_resources.html.erb

```diff
@@ -1 +1,5 @@
-<%= flot_chart('resource_graph', '', _("Number of Events"), resources_chart(params["range"].to_i.days.ago)) %>
+<%= react_component('AreaChart', id: 'resource_graph',
+                                 xAxisLabel: '',
+                                 yAxisLabel: _('Number of Events'),
+                                 data: resources_chart_data(params["range"].to_i.days.ago),
+                                 size: { height: 270 }) %>
```

Comment:
This (and the other one) used to be 250 previously, for me at least 270 causes the page to overflow.